### PR TITLE
Install WPCLI with docker and add the command to npm scripts

### LINKED ISSUE #828: Install WP CLI with the docker setup
WP cli is a handy tool that seems to be no longer included with the WordPress image.

### PR DESCRIPTION
**Pull request by dwainm**
Fixes #828

#### Changes proposed in this Pull Request

* Adds WP CLI installation to the docker build file.
* Adds a npm script shortcut for running it: `npm run wp....`
* Updates readme instruction to use npm script to interact with wp.

#### Testing instructions
- run `npm install`
- run `npm run up`
- Check that WPCLI was installed: `npm run wp -- --info `

-------------------

- [x] Tested on mobile (or does not apply)

### CODE REVIEW

File: docker/wordpress_xdebug/Dockerfile [hunk 1]
```diff
@@ -6,4 +6,7 @@ RUN pecl install xdebug \
 	&& echo 'xdebug.remote_autostart=1' >> $PHP_INI_DIR/php.ini \
 	&& docker-php-ext-enable xdebug
 RUN apt-get update \
-	&& apt-get install --assume-yes --quiet --no-install-recommends gnupg2 subversion mariadb-client
+	&& apt-get install --assume-yes --quiet --no-install-recommends gnupg2 subversion mariadb-client less
+RUN curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar \
+	&& chmod +x wp-cli.phar \
+	&& mv wp-cli.phar /usr/local/bin/wp
```
Comment (vbelolapotkov) [inline docker/wordpress_xdebug/Dockerfile:12]: There is an alternative method of setting up the cli using `wordpress:cli` container (see below). I'm not sure if it's better but it feels safer then downloading GH builds. What do you think?

```Dockerfile
# At the top of the file
FROM wordpress:cli AS cli
...
# After RUN command
COPY --from=cli /usr/local/bin/wp /usr/local/bin/wp
```

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "dev": "npm run up && npm run watch",
     "up": "docker-compose up --build --force-recreate -d",
     "down": "docker-compose down",
+    "wp": "docker-compose exec -u www-data wordpress wp",
```
Comment (vbelolapotkov) [inline package.json:36]: 👍

**Review by vbelolapotkov — APPROVED** [review]
Works and tests well 👍 There is a non-blocking comment on installing cli in dockerfile, leaving it up to you. Now with cli available in the container, do you think it's worth adjusting `cli` utility function in our setup scripts?

File: docker/wordpress_xdebug/Dockerfile (hunk 1 above)
Comment (dwainm) [reply docker/wordpress_xdebug/Dockerfile:12]: We can update that again if there's a need. This is more for daily usage.

**Pull request merged**